admin-core/build.gradle.kts

```diff
@@ -192,6 +192,14 @@ tasks {
     withType<JibTask> {
         dependsOn(processJibExtraDirs)
     }
+    val createDockerImage by registering(Exec::class) {
+        workingDir(projectDir)
+        commandLine(
+            "docker", "build",
+            "-t", "ghcr.io/drill4j/admin:$version",
```

Comment:
Move string with the registry/org/image names to a variable above (somewhere next to `var jvmArgs...`) and reuse it here and in `jib {` section
```
var fullImageTag = "ghcr.io/drill4j/admin"
...
"-t", "$fullImageTag:$version"

```